<ui version="4.0" >
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Contact Data</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>icons/contactlarge.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="contactNameLabel" >
        <property name="text" >
         <string>Contact Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="contactNameEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="extensionLabel" >
        <property name="text" >
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="extensionEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>151</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="saveButton" >
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
